From 0136a1f95380268f4635ccf9a4800c85d4bd8ed3 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Mon, 27 Nov 2017 14:11:35 +0800
Subject: [PATCH 2824/5242] MLK-16989-2 ARM64: dts: imx8mq: remove legacy
 fbdev dtbs

commit  5e569bea100b8c80cd9a53fc5dc101f0f73f0c1a from
https://source.codeaurora.org/external/imx/linux-imx.git

Remove below three dtbs related to legacy fbdev
on IMX8MQ platform:

   1. fsl-imx8mq-evk-4k.dtb
   3. fsl-imx8mq-evk-lcdif-dsi.dtb

since the fbdev drivers are not supported and
maintained anymore.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/Makefile             |    2 --
 .../arm64/boot/dts/freescale/fsl-imx8mq-evk-4k.dts |   24 ---------------
 .../dts/freescale/fsl-imx8mq-evk-lcdif-dsi.dts     |   31 --------------------
 3 files changed, 57 deletions(-)
 delete mode 100644 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-4k.dts
 delete mode 100644 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-dsi.dts

diff --git a/arch/arm64/boot/dts/freescale/Makefile b/arch/arm64/boot/dts/freescale/Makefile
index edceff7..3310e80 100644
--- a/arch/arm64/boot/dts/freescale/Makefile
+++ b/arch/arm64/boot/dts/freescale/Makefile
@@ -48,8 +48,6 @@ dtb-$(CONFIG_ARCH_FSL_IMX8QXP) += fsl-imx8qxp-lpddr4-arm2.dtb \
 				  fsl-imx8qxp-lpddr4-arm2-dsi-adv7535.dtb \
 				  fsl-imx8qxp-lpddr4-arm2-dsi-rm67191.dtb
 dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += fsl-imx8mq-evk.dtb \
-				 fsl-imx8mq-evk-4k.dtb \
 				 fsl-imx8mq-evk-drm.dtb \
-				 fsl-imx8mq-evk-lcdif-dsi.dtb \
 				 fsl-imx8mq-evk-m4.dtb \
 				 fsl-imx8mq-evk-pcie1-m2.dtb
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-4k.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-4k.dts
deleted file mode 100644
index 37072ac..0000000
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-4k.dts
+++ /dev/null
@@ -1,24 +0,0 @@
-/*
- * Copyright (C) 2016 Freescale Semiconductor, Inc.
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * as published by the Free Software Foundation; either version 2
- * of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- */
-
-#include "fsl-imx8mq-evk.dts"
-
-&dcss {
-	disp-mode = <97>;	/* #97: 3840x2160p@60Hz 16:9 */
-};
-
-&hdmi {
-	video-mode = <97>;
-};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-dsi.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-dsi.dts
deleted file mode 100644
index 0d89242..0000000
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-dsi.dts
+++ /dev/null
@@ -1,31 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * as published by the Free Software Foundation; either version 2
- * of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- */
-
-#include "fsl-imx8mq-evk.dts"
-
-&lcdif {
-	status = "okay";
-};
-
-&mipi_dsi {
-	status = "okay";
-};
-
-&dcss {
-	status = "disabled";
-};
-
-&hdmi {
-	status = "disabled";
-};
-- 
1.7.9.5

